vendor/github.com/elastic/go-libaudit/CHANGELOG.md

```diff
@@ -0,0 +1,64 @@
+# Change Log
```

Comment:
@ruflin, the way that I did it for the `libbeat/processors/kubernetes/vendor` is to remove common dependencies from the vendor.json.

there is no other way currently as this is not a first class govendor feature. we are just hacking it :)